website/docs/compute/customizing-functions/code-editor.md

```diff
@@ -0,0 +1,72 @@
+---
+pagination_prev: null
+---
+
+# Function editor
+
+The Compute code editor is available for you to embed in your application, 
+so your users can build their functions quickly and easily:
+
+![Compute editor containing a 'Hello' function](../../assets/editor-screen.png)
+
+The editor is hosted at `https://editor.suborbital.network`, and uses URL parameters to configure its connection to your builder service.
+
+You will host the Compute builder service in your cloud infrastructure with a configured DNS name. In order to use the editor, an HTTPS connection to the builder is required.
+
+To launch the editor, you can either [embed the editor in a frame](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe) within your own webpage, or launch it in a new tab.
+
+## Editor token
+
+To authenticate the editor for a specific user to edit their own functions, 
+[Compute's Administrative API provides an API to create an `editor token` ](https://suborbital-compute.readme.io/reference/gettoken)
+for a 
+given function.
```

Comment:
Weird line breaking here. Do we care?

Reply:
I think this is what prompted Oscar to file https://github.com/suborbital/docs/issues/162. I can fix this one manually in the meantime.